ci/pipeline.yml

```
@@ -1139,37 +1129,50 @@ jobs:
     plan:
       - get: weekly
         trigger: true
-      - get: golang-docker-image
+      - get: golang-registry-image
         trigger: true
-      - get: ubuntu-docker-image
+      - get: ubuntu-registry-image
         trigger: true
       - get: ruby-release
         trigger: true
         passed: [ bump-packages ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)